test/benchmark/core/UpdateMatrixWorld.js

```diff
@@ -11,7 +11,7 @@
 		var child = new THREE.Object3D();
 		child.position.copy( position );
 		child.scale.copy( scale );
-		child.rotation.copy( rotation );
+		child.quaternion.copy( rotation );
```

Comment:
Let's not write code that way.

Would you like to file a PR and rename `rotation` to `quaternion`?

Reply:
Yep.  Raised #25122